src/Abp.AutoMapper/AutoMapper/MultiLingualMapContext.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Abp.Configuration;
+
+namespace Abp.AutoMapper
+{
+    public class MultiLingualMapContext
+    {
+        public ISettingManager SettingManager { get; set; }
```

Comment:
Create a constructor that gets the setting manager as a required argument (and remove setter from the `SettingManager ` property). Otherwise, people may create a `MultiLingualMapContext` withot setting the SettingManager.